pkg/checks/external/main.go

```diff
@@ -596,14 +571,15 @@ func (ext *Checker) RunOnce() error {
 		return err
 	}
 
-	// waiting for all checker pods are gone...
+	// always run a cleanup when done
+	defer ext.cleanup()
```

Comment:
you have two `defer ext.cleanup()` in the same `RunOnce()` function in line 536,  i don't think it's needed since you specified this up top....?

Reply:
good catch! fixed and pushed